src/apps/mesoscale-explorer/data/generic/preset.ts

```diff
@@ -251,9 +251,10 @@ export async function createGenericHierarchy(plugin: PluginContext, file: Asset.
             }
             await build.commit();
 
-            const rootId = `${manifest.roots[0].id}:`;
+            // const rootId = `${manifest.roots[0].id}:`;
```

Comment:
we should pass rootId here since this is in a loop and we only want to update parts of the state tree